rfcat

```diff
@@ -15,21 +18,44 @@ logger = logging.getLogger(__name__)
 
 
 intro = """'RfCat, the greatest thing since Frequency Hopping!'
-
+Modified version for PandwaRF !
```

Comment:
```suggestion
```

Reply:
feel free to wrap in some logic to determine which dongle it is and print this for PandwaRF dongles.